src/attribute.cpp

```diff
@@ -19,8 +19,8 @@ bool Attribute::fetchId()
 {
     bool success = m_db.fetchId(uri());
     if (!success) {
-        qWarning() << "Failed to find Database entry for "
-                   << name() << ":" << uri();
+ //      qWarning() << "Failed to find Database entry for "
```

Comment:
Ditch this change from the pull request, please. 
Yes, it should be addressed, but properly - not like this.